src/main/kotlin/com/toasttab/pulseman/entities/CompileResult.kt

```diff
@@ -2,12 +2,10 @@ package com.toasttab.pulseman.entities
 
 import com.toasttab.pulseman.jars.JarLoader
 import com.toasttab.pulseman.pulsar.handlers.PulsarMessageClassInfo
-import javax.script.ScriptEngine
 
 class CompileResult(
     val code: String,
     val classToGenerate: PulsarMessageClassInfo,
     val jarLoader: JarLoader,
-    val engine: ScriptEngine,
```

Comment:
There are bugs brought in with the latest 1.92+ version of kotlin scripting reusing the ScriptEngine, it doesn't happen at runtime, so I'm going to remove and just recreate it to workaround.